docs/application/web/guides/multimedia/media-controller.md

```diff
@@ -260,8 +271,125 @@ Server can set age rating for current playback. Client can access this rating (r
     }
     ```
 
+## Managing playlists on server side
+
+To manage the media controller playlists in your server application, you must learn to create, save, delete playlists.
+
+1. Create a media controller playlist using the `createPlaylist()` method:
+
+   ```
+   var playlist = mcServer.createPlaylist("testPlaylistName");
+   ```
+
+2. Adding item to playlist is available using `addItem()` method:
```

Comment:
Suggest rephrasing to:

2. Add item to the playlist using the `addItem()` method:

Reply:
I rephrased to
**_To add item to playlist use `addItem()` method_**